Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="index.css">

```

Comment:
The link tag should have a closing forward slash, like this: <link rel='stylesheet' href='index.css'/>